src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPasswordWrapper.java

```diff
@@ -79,6 +79,14 @@ public void setPasswordEntries(EnvInjectPasswordEntry[] passwordEntries) {
 
     @Override
     public Environment setUp(AbstractBuild build, Launcher launcher, BuildListener listener) throws IOException, InterruptedException {
+        EnvInjectLogger logger = new EnvInjectLogger(listener);
+        if (isInjectGlobalPasswords()) {
+            logger.info("Inject global passwords.");
```

Comment:
This message makes no sense now since it does not identify the stage in DecorateLogger()